lib/cassandra/cql.lua

```diff
@@ -102,12 +102,14 @@ local ERRORS            = {
   READ_TIMEOUT          = 0x1200,
   READ_FAILURE          = 0x1300,
   FUNCTION_FAILURE      = 0x1400,
+  WRITE_FAILURE         = 0x1500,
   SYNTAX_ERROR          = 0x2000,
   UNAUTHORIZED          = 0x2100,
   INVALID               = 0x2200,
   CONFIG_ERROR          = 0x2300,
   ALREADY_EXISTS        = 0x2400,
   UNPREPARED            = 0x2500,
+  UNKNOWN               = -1,
```

Comment:
I don't think this constant is necessary. The binary protocol guarantees us we will be receiving an error code already.

Reply:
Nice! Will remove then